nixos/modules/services/networking/solanum.nix

```
@@ -0,0 +1,132 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) mkEnableOption mkIf mkOption singleton types;
+  inherit (pkgs) coreutils solanum;
```

```suggestion
  inherit (pkgs) solanum;
```

Or where do you need coreutils?